fasthtml/js.py

```diff
@@ -12,16 +13,41 @@ def MarkdownJS(sel='.marked'):
     src = "proc_htmx('%s', e => e.innerHTML = marked.parse(e.textContent));" % sel
     return Script(marked_imp+src, type='module')
 
-def KatexMarkdownJS(sel='.marked', katex_tags='$'):
-    right_tags = '\\$' if katex_tags=='$' else '\\]'
+def KatexMarkdownJS(sel='.marked', inline_delim='$', display_delim='$$', math_envs=None):
+    math_envs = math_envs or ['equation', 'align', 'gather', 'multline']
+    env_list = ','.join(f"'{env}'" for env in math_envs)
+
     src = """
```

Comment:
```suggestion
    src = r"""
```

IIIUC you should be able to remove the doubled slashes if you change this